src/main/java/fr/esir/jxc/services/UserService.java

```diff
@@ -0,0 +1,5 @@
+package fr.esir.jxc.services;
+
+public interface UserService {
+    void sendModifiedPasswordEvent(String email, String newPassword);
```

Comment:
```suggestion
    void sendPasswordUpdatedEvent(String email, String newPassword);
```

Cf contrats d'interface : https://docs.google.com/document/d/1z3HvGJmQf1_F_94hePpaABpPoCmRO45ReeygwAdt7hw/edit?usp=sharing